src/framework/core/js/Fluid.js

```diff
@@ -826,6 +826,28 @@ var fluid = fluid || fluid_3_0_0;
         return isFinite(string) && ((string % 1) === 0) ? Number(string) : NaN;
     };
 
+    /**
+     * Derived from Sindre Sorhus's round-to node module ( https://github.com/sindresorhus/round-to ).
+     * License: MIT
+     *
+     * Rounds the supplied number to at most the number of decimal places indicated by the scale, omiting any trailing 0s.
+     * Numbers are rounded away from 0 (i.e 0.5 -> 1, -0.5 -> -1).
+     * If the scale is invalid (i.e falsey, not a number, negative value), it is treated as 0.
+     * If the scale is a floating point number, it is rounded to an integer.
+     *
+     * @param {Number} num - the number to be rounded
+     * @param {Number} scale - the maximum number of decimal places to round to.
+     * @return {Number} The num value rounded to the specified number of decimal places.
+     */
+    fluid.roundToDecimal = function (num, scale) {
```

Comment:
I think it's worth preserving all of sindre's original implementation - it doesn't cost much for the extra flexibility - that is, to expose the possibility for round, ceil and floor rounding, and then to also expose these options in the transformSpec of fluid.transforms.numberToString . We can preserve the actual fixtures (num, scale) for our test cases and add in extra "expected" values to illustrate the results of different rounding.